app/presenters/track.rb

```diff
@@ -0,0 +1,39 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+        @track = Trackler.tracks[track_id]
+      end
+
+      def method_missing(method, *args)
+        @track.send(method, *args)
+      end
+
+      def docs
+        track_docs = @track.docs("/api/v1/tracks/%s/images/docs/img" % @track_id)
+
+        track_docs.each_pair do |topic_name, topic_content|
+          track_docs[topic_name] = if topic_content.present?
```

Comment:
Good call! I made that change as part of [this commit](https://github.com/exercism/exercism.io/pull/3454/commits/ba046e7aa4890e201aa998002b9cc1e6e481e73c).